Mathlib/Data/List/Basic.lean

```diff
@@ -786,6 +791,11 @@ theorem ext_get?_iff' {l₁ l₂ : List α} : l₁ = l₂ ↔
     ∀ n < max l₁.length l₂.length, l₁.get? n = l₂.get? n :=
   ⟨by rintro rfl _ _; rfl, ext_get?'⟩
 
+/-- If two lists are the same length and get! is the same on all indices, the lists are equal. -/
```

Comment:
```suggestion
/-- If two lists are the same length and `List.get!` is the same on all indices, the lists are equal. -/
```

Reply:
Its `_[_]!`, not `get!` (I've changed this while cherry-picking). I'll push the update in a minute.